org.adempiere.base.process/src/org/compiere/process/M_PriceList_Create.java

```diff
@@ -295,7 +296,9 @@ else if (M_Pricelist_Version_Base_ID == 0) {
 						+ " INNER JOIN M_DiscountSchemaLine dl ON (dl.M_DiscountSchemaLine_ID=?) "	//	#3
 						//
 						+ "WHERE EXISTS (SELECT * FROM T_Selection s WHERE s.AD_PInstance_ID=? AND po.M_Product_ID=s.T_Selection_ID)"	//	#4
-						+ " AND po.IsCurrentVendor='Y' AND po.IsActive='Y'";
+						+ " AND (dl.C_BPartner_ID IS NULL AND po.IsCurrentVendor='Y'"

```

Comment:
This change doesn't seem equivalent:
from:
AND po.IsCurrentVendor='Y'
to:
AND (dl.C_BPartner_ID IS NULL AND po.IsCurrentVendor='Y'
OR po.C_BPartner_ID=dl.C_BPartner_ID AND (dl.IsIgnoreIsCurrentVendor='Y' OR po.IsCurrentVendor='Y'))"

Seems like a change on the previous feature, where is checking IsCurrentVendor independently of the C_BPartner configuration.

Reply:
Hi Carlos, @globalqss 
the part behaves exactly as before if IsIgnoreIsCurrentVendor is N. If bpartner is null then IsCurrentVendor defines the result and if bpartner is not null it defines result also as long as IsIgnoreIsCurrentVendor doesn't do it. So it's independent of bpartner but not independent of IsIgnoreIsCurrentVendor